docs/package_workflows.md

```diff
@@ -1,40 +1,54 @@
-HTML Logic
+# HTML Logic
 
 ``` mermaid
 flowchart TD
-    A[Start] --> B[Parse Arguments]
-
-
-    B --> X[Read Config]
-    X --> Y[Read Input File]
-    Y --> AA[Validate Input File]
-
-    AA -->|Valid| AB[Soupify Input File]
-    AA -->|Invalid| AC[Handle Error] --> Z
-
-    AB --> AD[Clean Text]
-    AD --> AE[Extract Keywords]
-    AE --> AF[Extract Title]
-    AF --> AG[Extract Sections]
-    AG --> AH[Extract Text]
-    AH --> AI[Set Unknown Section Headings]
-    AI --> AJ[Handle HTML]
-    AJ --> AK[Merge Table Data]
-    AK --> AL[Convert to BioC]
-
-    AL --> AM[Output BioC JSON]
-    AL --> AN[Output BioC XML]
-    AL --> AO[Output Tables to BioC JSON]
-    AL --> AP[Output Abbreviations to BioC JSON]
-    AL --> AQ[Output to JSON]
-    AL --> AR[Output to Dictionary]
-
-    AM --> Z
-    AN --> Z
-    AO --> Z
-    AP --> Z
-    AQ --> Z
-    AR --> Z
-
-    Z[End]
+
+    classDef Autocorpus  fill:lightblue, stroke:black;
+    classDef TODO fill:orange, stroke:black;
+    classDef __main__ fill:lightgreen, stroke:black;
+
+    subgraph Legend[Classes/Files]
+        legend_autocorpus[Autocorpus]:::Autocorpus
+        legend_main[__ main __.py]:::__main__
+        legend_todo[placeholder function]:::TODO
+    end
+
+
+    A[Start]:::__main__ --> B[Parse Arguments]:::__main__
+    B --> read_file_structure[read file structure]:::__main__
+
+    read_file_structure --> X[Read Config]:::Autocorpus
+    X --> validate_config
+
+    validate_config:::TODO --> Y[Read Input File]:::Autocorpus
+    Y --> AA[Validate Input File]:::TODO
+    AA -->|Valid| AB[Soupify Input File
+    >Autocorpus:42]:::Autocorpus
```

Comment:
We will likely be refactoring some of the code, so these line numbers will become out-dated. It's probably best to name the methods. Like so:
```suggestion
    >Autocorpus:__soupify_infile]:::Autocorpus
```